src/sass/components/_card.scss

```diff
@@ -48,8 +49,8 @@
     }
 
     &--outline {
-      border: 1px solid $selected-color;
-      border-radius: $radius-sm3;
+      border: 1px solid $link-color;
+      border-radius: 3px;
```

Comment:
Please test with `npm test` -- I don't think specifying 3px here is allowed by eslint.